cogs/StaffAppsBackoffice.py

```diff
@@ -0,0 +1,966 @@
+        """
+        self.children[2].disabled = False  # enables the "mark as spam" button
+        self.children[3].disabled = False  # enables the "deny" button
+        self.children[4].disabled = False  # enables the "advance to next stage" button
+        # page one back button check
+        if self.cur_page != 1:
+            self.children[0].disabled = False
+        # second opinion check
+        if self.data[self.cur_page - 1][2] == 1:
+            self.children[5].disabled = False
+        else:
+            self.children[5].disabled = True
+
+        # long application check
+        if (
+            len(self.data[self.cur_page - 1][8]) > 1024
+            or len(self.data[self.cur_page - 1][9]) > 1024
+        ):
+            self.children[8].disabled = False
+        else:
+            self.children[8].disabled = True
+
+        self.db = sqlite3.connect("util/database.sqlite")
+        self.cursor = self.db.cursor()
+        statusname = self.cursor.execute(
+            "select s.description from application a join status s on a.status = s.statusID where appId = ?;",
+            (self.data[self.cur_page - 1][0],),
+        ).fetchone()[0]
+        status_int = self.data[self.cur_page - 1][2]
+        # check likes and dislikes and disable buttons if necessary
+        like_amount = len(
+            self.cursor.execute(
+                "select likes from like where uid = ? and appId = ?;",
+                (self.author.id, self.data[self.cur_page - 1][0]),
+            ).fetchall(),
+        )
+        like_list = [
+            x[0]
+            for x in self.cursor.execute(
+                "select likes from like where appId = ?;",
+                (self.data[self.cur_page - 1][0],),
+            ).fetchall()
+        ]
+        likes = like_list.count(1)
+        dislikes = like_list.count(0)
+        if like_amount > 0:
+            self.children[6].disabled = True
+            self.children[7].disabled = True
+        else:
+            self.children[6].disabled = False
+            self.children[7].disabled = False
+        self.db.close()
+        title = f"Staff applications page {self.cur_page}/{len(self.data)}"
+        description = f"Application ID: **{self.data[self.cur_page-1][0]}**\nApplicant name: **{self.data[self.cur_page-1][3]}**\nApplicant ID: **{self.data[self.cur_page-1][1]}**\n"
+        fields = [
+            ["Status", statusname, False],
+            ["first name", self.data[self.cur_page - 1][4], False],
+            ["Time zone", self.data[self.cur_page - 1][6], False],
+            ["Hours available per week", self.data[self.cur_page - 1][7], False],
+            ["Why staff?", self.data[self.cur_page - 1][8][:1024], False],
+            ["Contribute reason", self.data[self.cur_page - 1][9][:1024], False],
+            [
+                "Submission time",
+                datetime.fromtimestamp(int(self.data[self.cur_page - 1][10])),
```

Comment:
datetime.fromtimestamp() takes an EPOCH timestamp as argument, EPOCH timestamps are the time in seconds elapsed since 00:00:00 UTC on 1 January 1970. This object is already timezone-aware at the time of creation.